Add Backup / Restore docs (#2072)
codenvy/che-docs: modules/administration-guide/partials/assembly_backup-and-disaster-recovery.adoc

:context: backup-and-disaster-recovery

{prod} Operator can create backups of {prod-short} instances and restore them from a backup snapshot if needed. The following chapter describes ways of preparing such backups and their use in the follow-up recovery phase:

> * xref:managing-backups-through-cli.adoc[]

Reviewer: ```suggestion
* xref:managing-backups-using-cli.adoc[]
```

> * xref:managing-backups-through-custom-resources.adoc[]

Reviewer: ```suggestion
* xref:managing-backups-using-custom-resources.adoc[]
```

:context: backup-and-disaster-recovery

{prod} Operator can create backups of {prod-short} and restore from a backup snapshot if needed.

> The following ways available to create backups and do recovery:

Reviewer: ```suggestion
The following ways are available to create backups and do recovery:
```

* xref:managing-backups-through-cli.adoc[]
* xref:managing-backups-through-custom-resources.adoc[]

[WARNING]
====

> Standard backup mechanism of {prod} does not back up content of users' workspaces.

Reviewer: ```suggestion
The standard backup mechanism of {prod} does not back up the content of users' workspaces.
```

> To preserve local changes, see xref:persistent-volumes-backups.adoc[]

Reviewer: ```suggestion
To preserve local changes, see xref:persistent-volumes-backups.adoc[].
```

====

[CAUTION]
====

> Backup snapshots are bind to specific cluster and must be used only on it.

Reviewer: ```suggestion
Backup snapshots are bound to a specific cluster and can only be used on it.
```
